packages/logs/src/boot/preStartLogs.ts

```diff
@@ -23,9 +24,11 @@ export function createPreStartStrategy(
   const bufferApiCalls = new BoundedBuffer<StartLogsResult>()
   let cachedInitConfiguration: LogsInitConfiguration | undefined
   let cachedConfiguration: LogsConfiguration | undefined
+  const trackingConsentState = createTrackingConsentState()
+  trackingConsentState.observable.subscribe(tryStartLogs)
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏is it ok to trigger multiple startLogs if the consent API is called multiple times with the granted value?

Reply:
It is not OK to call `startLogs` multiple time. Do you see any scenario where this can happen?

Reply:
```js
DD_LOGS.setTrackingConsent('granted')
DD_LOGS.setTrackingConsent('granted')
```

Probably not on purpose but instrumentation mistakes can happen (like multiple init).

Reply:
It calls `tryStartLogs` multiple times, but not `startLogs`, right?

Reply:
oh, it actually not even calls `tryStartLogs` multiple times, since two different trackingConsent components are used before and after start.